csharp-ovh/Client/Client.Put.cs

```diff
@@ -68,9 +68,9 @@ public Task<string> PutAsync(string target, string data, bool needAuth = true)
         /// <param name="data">Json data to send as body</param>
         /// <param name="needAuth">If true, send authentication headers</param>
         /// <returns>API response deserialized to T by JSON.Net</returns>
```

Comment:
Please add the documentation for the timeout param, stating that it overrides the client's one (and/or that it defaults to it)

Reply:
Documentation added on all Async calls